src/components/metadata/field.js

```diff
@@ -2,14 +2,18 @@
 
 const React = require('react')
 const { Editable } = require('../editable')
+const { DragSource, DropTarget } = require('react-dnd')
 const { FormattedMessage } = require('react-intl')
 const { blank, noop, pluck, URI } = require('../../common/util')
 const { IconLock, IconWarningSm } = require('../icons')
 const cx = require('classnames')
-const { TYPE } = require('../../constants')
+const { TYPE, DND } = require('../../constants')
 const { getMetadataCompletions } = require('../../selectors')
 const { auto } = require('../../format')
-const { bool, func, number, oneOfType, shape, string } = require('prop-types')
+const { getEmptyImage } = require('react-dnd-electron-backend')
+const { shapes } = require('../util')
```

Comment:
Unused import. (`npm run lint` will [catch those](https://travis-ci.org/tropy/tropy/jobs/578237842#L1354) for you, but it's best to use an eslint plugin in your editor)